master/buildbot/test/fake/docker.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, base_url):
     def images(self):
         return self._images
 
-    def start(self, *args, **kwargs):
+    def start(self, instance):
```

Comment:
I think argument should be called `container`: https://docker-py.readthedocs.org/en/latest/api/#start